Makefile

```diff
@@ -29,7 +29,7 @@ ruff:
 	ruff format .
```

Comment:
I guess this is nice for `make ruff`, so you still get formatting even if you have an error?